### ISSUE TITLE
FE7: Strange Text Issue When Choosing the Storyline?

### ISSUE THREAD

**Issue by LancetheGallade1500** [lushen124/Universal-FE-Randomizer#243]
I did an ordinary Recruitment Randomizer, nothing else. When I hover over Louise (Lyn's replacement), the chapter title _appears_ normal for a bit, but as soon as Louise starts doing her animation, the text tilts.

![YuneTest 2020-04-13 17 36 22](https://user-images.githubusercontent.com/48635148/79173839-9c1a3980-7dad-11ea-90f8-b3d2f7bd7536.png)

(I heard that I'm not supposed to share links to ROMs, so I converted the first page of the html file to a pdf in case anyone wants to check if they get the same issue.)

[YuneTest pages-1.pdf](https://github.com/lushen124/Universal-FE-Randomizer/files/4472531/YuneTest.pages-1.pdf)

**Comment by lushen124** [comment]
Yes, this is a known issue. I'm not quite sure why the text rotates there, and it only happens with some classes. As it is primarily visual in nature, it's a bit of a lower priority for the time being. I'm assuming some classes trigger rotation at some point in their animation and that's, for whatever reason, tied to the text graphics at the top.
